src/SDKs/Azure.ApplicationModel.Configuration/data-plane/README.md

```diff
@@ -1,7 +1,12 @@
 # Overview 
 
 Azure.ApplicationModel.Configuration is a component of the .NET Azure SDK. 
-It provides APIs for storing and retrieving application settings.
+It provides APIs for Microsoft [Azure's App Configuration Service](https://docs.microsoft.com/en-us/azure/azure-app-configuration/).
+
+Developers can use this SDK to interact with their [Configuration Stores](https://docs.microsoft.com/en-us/azure/azure-app-configuration/quickstart-dotnet-core-app#create-an-app-configuration-store) in the following ways:
+- Perform basic reads, writes, updates, and deletes
```

Comment:
I would add "... of application configuration settings". Also, it might be worth adding a sentence or two above, in the overview, describing the concept of a configuration setting. My worry is that if the reader just clicks on the link to the service, they will not see anything about settings. the service docs talk about key-value pairs, don't they?

Reply:
They do... I guess there is a difference on terminology there. To clarify I added "configuration settings or key-value"